src/main/webapp/resources/js/components/samples/components/SampleMetadata.tsx

```diff
@@ -53,27 +53,34 @@ export function SampleMetadata() {
     );
   }, []);
 
-  const removeMetadata = (field, entryId) => {
+  const removeMetadata = (field: string, entryId: number) => {
     removeSampleMetadata({
       projectId,
       field,
       entryId,
     })
-      .then(({ data }) => {
-        notification.success({ message: data.message });
+      .unwrap()
+      .then(({ message }) => {
```

Comment:
Type please

Reply:
Updated in eb010bbe25ec4539fd70152d7567165863866208